llarp/dns/question.cpp

```diff
@@ -64,6 +64,12 @@ namespace llarp
       return (qname == "localhost.loki." or llarp::ends_with(qname, ".localhost.loki."));
     }
 
+    bool
+    Question::HasSubdomains() const
+    {
+      return qname.size() >= 3;
```

Comment:
I don't understand this.

Reply:
sub.main.loki has 3 parts, sub, main and loki